config/gulp/build.js

```diff
@@ -85,13 +85,8 @@ gulp.task('build-uswds-if-needed', function () {
       ));
     }
 
-    //  "federalist": "npm install --dev && gulp build && fractal build",
     const sharedOpts = { stdio: 'inherit', cwd: uswdsDir };
-    return spawnP('npm', [ 'install', '--only=dev', '--ignore-scripts' ],
-                  sharedOpts)
-      .then(() => spawnP('gulp', [ 'build' ], sharedOpts))
-      .then(() => spawnP('./node_modules/.bin/fractal', [ 'build' ],
-                         sharedOpts));
+    return spawnP('npm', [ 'run', 'federalist' ], sharedOpts);
```

Comment:
When just running `npm run federalist` the first npm install command would work, but then any gulp commands after that would break because gulp would switch the current directory and would try and load config from the docs folder rather then the uswds folder. So it would fail because it couldn't find `gulp release`